docs/source/conf.py

```diff
@@ -348,14 +348,25 @@ def inject_weight_metadata(app, what, name, obj, options, lines):
             lines.append("")
 
 
-def generate_weights_table(module, table_name, metrics, include_pattern=None, exclude_pattern=None):
+def generate_weights_table(module, table_name, metrics, include_patterns=None, exclude_patterns=None):
+    """
+
+    module: corresponding torchvision module
+    table_name: Name of table generated.
+    metrics: Metrics from weights enum to be used.
+    include_patterns: List of patterns to include
+    exclude_patterns: List of patterns to exclude
+
+    Generates table file and saves it to generated folder.
+
+    """
```

Comment:
I would suggest to  remove this docstring, it doesn't add much additional info to the parameter names. A good description would require a much longer docstring, but I don't feel like it's worth doing this right now.

Reply:
Removed, and changed the subsection. I hope the new layout is what you asked :)